File: module_main/src/main/java/com/yyxnb/module_main/ui/MainActivity.java
package com.yyxnb.module_main.ui;

import android.support.v4.app.Fragment;

import com.yyxnb.common_base.base.ContainerActivity;

/**
 * @author yyx
 */
public class MainActivity extends ContainerActivity {
    @Override
    public Fragment initBaseFragment() {
        return new MainHomeFragment();
    }

}

File: module_main/src/main/java/com/yyxnb/module_main/ui/MainActivity.kt
package com.yyxnb.module_main.ui

import androidx.fragment.app.Fragment
import com.yyxnb.common_base.base.ContainerActivity

/**
 * @author yyx
 */
class MainActivity : ContainerActivity() {

    override fun initBaseFragment(): Fragment? {
        return MainHomeFragment()
    }
}